lib/galaxy/datatypes/tabular.py

```diff
@@ -320,7 +320,11 @@ class Tabular(TabularData):
     file_ext = "tabular"
 
     def get_column_names(self, first_line=None):
-        return None
+        if first_line.startswith("#"):
```

Comment:
this will break if `first_line` is `None`, which, I assume, it could be given the default argument.